app/services/user_rating_service.rb

```
@@ -59,7 +59,7 @@ def update_database_columns(participant_ids, new_participant_ratings, new_partic
       end
       UserRating.import userratings
       unless @temporary
-        @round.update_attribute :calculated_permanent, true
+        @round.update(calculated_permanent: true)
```

It's strongly recommended to use `update!` instead of `update` to not swallow errors silently.